### ISSUE TITLE
cc-release: Assertion error on boot

### ISSUE THREAD

**Issue by jtobin** [urbit/urbit#1279]
Per #1274, booting without specifying a pill bails after reporting a failed assertion on Debian 10:

```
jtobin@pollux:~/urbit$ urbit -w nidsut-tomdun -k $KEYFILE
~
urbit 0.8.0
boot: home is /home/jtobin/urbit/nidsut-tomdun
loom: mapped 2048MB
lite: arvo formula 3685298
lite: core b41b6d6
lite: final state b41b6d6
Assertion '0 != u3_Host.ops_u.url_c' failed in vere/daemon.c:555

bail: oops
bailing out
urbit: noun/manage.c:663: u3m_bail: Assertion `0' failed.
Aborted (core dumped)
```

@joemfb's commentary:

> That assertion failure should never be happening -- it points to some kind of error in the argument parsing in daemon/main.c.

**Comment by jfranklin9000** [comment]
Maybe on topic (not specifying a pill):
```
/nix/store/awpq8namkrza0gm0v4kzlxh4264lncvx-urbit/bin/urbit -c mycomet
~
urbit 0.8.0
boot: home is /Volumes/sensitive/URBIT/urbit-cc-release/mycomet
loom: mapped 2048MB
lite: arvo formula 3685298
lite: core b41b6d6
lite: final state b41b6d6
boot: downloading pill https://bootstrap.urbit.org/urbit-0.8.0.pill
error fetching https://bootstrap.urbit.org/urbit-0.8.0.pill: HTTP 403
```

**Comment by jfranklin9000** [comment]
_(apologies for highjacking the thread)_
```
/nix/store/awpq8namkrza0gm0v4kzlxh4264lncvx-urbit/bin/urbit -B bin/solid.pill -c mycomet
~
urbit 0.8.0
boot: home is /Volumes/sensitive/URBIT/urbit-cc-release/mycomet
loom: mapped 2048MB
lite: arvo formula 3685298
lite: core b41b6d6
lite: final state b41b6d6
boot: loading pill bin/solid.pill
boot: mining a comet. May take up to an hour.
If you want to boot faster, get an Azimuth point.
boot: found comet ~lavfex-wicdef-lislud-mapsel--sonten-tabsur-dotnul-marzod
boot: downloading azimuth snapshot from https://bootstrap.urbit.org/urbit-0.8.0.snap
error fetching https://bootstrap.urbit.org/urbit-0.8.0.snap: HTTP 403
```
I understand why `https://bootstrap.urbit.org/urbit-0.8.0.snap` doesn't exist; is there a way to complete this step?

**Comment by joemfb** [comment]
Yeah, you can use just download and use `urbit-0.7.4.snap`. Specify it with `-E path/to/snapshot`.

**Comment by jtobin** [comment]
Closed in #1282.
